client/packages/system/src/Patient/Encounter/DetailModal.tsx

```diff
@@ -42,15 +42,14 @@ const useUpsertEncounter = (
 export const EncounterDetailModal: FC = () => {
   const patientId = usePatient.utils.id();
 
-  const { current, documentName, programType, documentType, reset } =
-    usePatientModalStore();
+  const { current, document, programType, reset } = usePatientModalStore();
```

Comment:
I refactored the patient modal store slightly, since I ended up with three `setDocument...` methods!